src/main/java/com/google/transit/realtime/GtfsRealtimeExtensions.java

```diff
@@ -40,5 +40,6 @@ public static void registerExtensions(ExtensionRegistry registry) {
     registry.add(GtfsRealtimeOneBusAway.obaEntitySelector);
     registry.add(GtfsRealtimeOneBusAway.obaStopTimeUpdate);
     registry.add(GtfsRealtimeMNR.mnrStopTimeUpdate);
+    // warning: cannot add both GtfsRealtimeMNR.mnrStopTimeUpdate and GtfsRealtimeLIRR.MtaStopTimeUpdate.track to the same registry
```

Comment:
IMHO configuration of this needs to somehow be exposed by the parameters of this method, and the Javadocs for the method need to change.  It currently says `Adds all known GTFS-realtime extension messages to the specified extension registry.`, which if I understand correctly is no longer true from the above comment.  Also, additional info as to why they can't be registered at the same time would be helpful.

Reply:
Thanks @barbeau, good point. I added some additional documentation and allowed for some configuration. MNR and LIRR are both using extension ID 1005 with different extensions, which is technically invalid, but is necessary to read both of their realtime feeds with the track extension. If this is rectified in the future, we can revert to the earlier, simpler behavior of this method.

Reply:
Thanks for updating the code, that looks better.

Yeah, that doesn't sound right re: shared extension.  According to the [official extension registry](https://github.com/google/transit/blob/master/gtfs-realtime/CHANGES.md#extension-registry) MRN owns the `1005` ID.  So LIRR stole theirs?  Any chance we can get LIRR to apply for their own extension ID?

Reply:
I think it's just a misunderstanding, especially since the extensions are intended to do the same thing (specify a track for a StopTimeUpdate). I think we can get LIRR to adopt the MNR version, but not necessarily in the short term...